traits/tests/test_enum.py

```diff
@@ -34,8 +39,22 @@ def _get_valid_models(self):
         return ["model1", "model2", "model3"]
 
 
-class EnumListExample(HasTraits):
+class CustomCollection(Collection):
+
+    def __init__(self, *data):
+        self.data = tuple(data)
+
+    def __len__(self):
+        return len(self.data)
+
+    def __iter__(self):
+        return iter(self.data)
+
+    def __contains__(self, __x: object):
```

Comment:
What's the reason for the double-underscore prefix for `x`?

Also, I'd suggest not introducing typing annotations into the core of Traits just yet. We may want to do that at some point, but I think it needs to be an explicit decision by the ETS maintainers, and it needs to be done fairly consistently: it doesn't make sense to have only the occasional type annotation.

Reply:
This was some unintentional auto generated code, changed it.